specs/pvs/corla.pvs

```diff
@@ -243,8 +258,10 @@ BEGIN
 %%% unique name.
 % @review CDOS will there be one risk limit? If more than one, then
 % system must provide a way to map contests to risk limits.
```

Comment:
Yes, I'll remove that @review and other ones that have now been addressed by CDOS.